schema/spacewalk/upgrade/susemanager-schema-4.3.3-to-susemanager-schema-4.3.4/002-remove-minion_pillars_func.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS minion_pillars;
```

Comment:
With uyuni released with susemanager-schema version 4.3.4 this should now be 4.3.4-4.3.5.

Reply:
oh right, I forgot to check after the rebase.